src/main/java/org/jenkinsci/plugins/gitclient/GitClient.java

```diff
@@ -435,6 +435,12 @@
      */
     void clean() throws GitException, InterruptedException;
 
+    /**
+     * clean_.
+     *
+     * @return a {@link org.jenkinsci.plugins.gitclient.CleanCommand} object.
+     */
+    CleanCommand clean_(); // can't use 'clean' as legacy IGitAPI already define this method
```

Comment:
Not sure how 'clean2' is more meaningful. I'm just following the naming convention used in this class. Like clone_, fetch_, init_ and revList_.

E.g.:
FetchCommand fetch_(); // can't use 'fetch' as legacy IGitAPI already define this method